packages/wonder-blocks-dropdown/docs.md

```diff
@@ -0,0 +1,51 @@
+Temporary views of ActionItem, SelectItem, and SeparatorItem. Will be replaced
+by higher-level menu components when they are merged.
+
+```js
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+    },
+    darkBackgroundWrapper: {
+        background: "black",
+        padding: 10,
+    },
+    strutLike: {
+        width: 8,
+        height: 8,
+    },
+    sepContainer: {
+        width: 200,
+        height: 10,
+    },
+});
+<View style={[styles.darkBackgroundWrapper]}>
+    <View style={[styles.row]}>
+        <ActionItem label={"Go to KA"} href={"https://khanacademy.org"} />
+        <View style={[styles.strutLike]} />
```

Comment:
One alternative would be to add a `style` prop to `ActionItem` and then use `marginRight`, `marginBottom`, but since this is going away in future PRs it's fine as is.